_data/wildfly-categories.yaml

```diff
@@ -0,0 +1,140 @@
+---
+categories:
+  - name: Agroal
+    id: agroal
+    description: Agroal Data Source
+  - name: Batch
+    id: batch
+    description: Jakarta EE Batch
+  - name: Bean Validation
+    id: bean-validation
+    description: Jakarta EE Bean Validation
+  - name: Bootable Jar
+    id: bootable-jar
+    description: Bootable Jar
+  - name: Command Line Interface (CLI)
+    id: cli
+    description: Command Line Interface
+  - name: Cloud
+    id: cloud
+    description: Developing and Running a WildFly application on the cloud
+  - name: User Experience
+    id: user-experience
+    description: Impact how the user experiences working with WildFly
+  - name: Clustering
+    id: clustering
+    description: Clustering, including Infinispan and JGroups
+  - name: Concurrency
+    id: concurrency
+    description: Concurrency
+  - name: Console
+    id: console
+    description: Console
+  - name: WildFly Controller
+    id: controller
+    description: WildFly Controller
+  - name: WildFly Core
+    id: core
+    description: WildFly Core
+  - name: Jakarta EE
+    id: ee
+    description: Jakarta EE
+  - name: Jakarta EJB
+    id: ejb
+    description: Jakarta Entreprise Java Beans (EJB)
+  - name: WildFly Elytron
+    id: elytron
+    description: WildFly Elytron
+  - name: Embedded
+    id: embedded
+    description: Embedded
+  - name: IIOP
+    id: iiop
+    description: Internet InterORB Protocol
+  - name: Jakarta RS
+    id: jaxrs
+    description: Jakarta RESTful Web Services
+  - name: JCA
+    id: jca
+    description: Jakarta Connectors Architecture
+  - name: JDR
+    id: jdr
+    description: JBoss Diagnostic Reporter
+  - name: JPA
+    id: jpa
+    description: Jakarta Persistence
+  - name: JSF
+    id: jsf
+    description: Jakarta Faces
+  - name: Logging
+    id: logging
+    description: Logging
+  - name: Jakarta Mail
+    id: mail
+    description: Mail
+  - name: Management
+    id: management
+    description: Management
+  - name: Maven
+    id: maven
+    description: Maven Tooling & Ecosystem
+  - name: Messaging
+    id: messaging
+    description: Messaging, including Jakarta Messaging (JMS)
+  - name: Eclipse MicroProfile
+    id: microprofile
+    description: Eclipse MicroProfile
+  - name: Migration
+    id: migration
+    description: Migration
+  - name: Mod_cluster
+    id: mod_cluster
+    description: mod_cluster
+  - name: Modules
+    id: modules
+    description: JBoss Modules & Java Platform Module System (JPMS)
+  - name: JBoss MSC
+    id: msc
+    description: JBoss MSC
+  - name: Observability
+    id: observability
+    description: Observability
+  - name: OpenShifft
```

Comment:
OpenShift

I guess two 'f's is better than none!

Reply:
and it's better than none (once I sent a mail to a wider audience about `OpenShit`, damn typo! :)

Reply:
yup, I did the "no f" typo in a mailing list once :)